src/Map/src/Bridge/Leaflet/assets/dist/map_controller.js

```diff
@@ -38,14 +38,21 @@ class map_controller extends AbstractMapController {
         const { position, title, infoWindow, extra, rawOptions = {}, ...otherOptions } = definition;
         const marker = L.marker(position, { title, ...otherOptions, ...rawOptions }).addTo(this.map);
         if (infoWindow) {
-            this.createInfoWindow({ definition: infoWindow, marker });
+            if (infoWindow.opened) {
+                this.createInfoWindow({ definition: infoWindow, marker });
+            }
+            else {
+                marker.on('click', () => {
+                    this.createInfoWindow({ definition: infoWindow, marker, onMarkerClick: true });
+                });
```

Comment:
Does not seem a good idea to run every time.

Reply:
What do you mean?

Reply:
"createInfoWindow" will be called on every "click" ?

Reply:
Ah, yes you're right.

Hum, I may have better ideas to rewrite the PR, but I don't have time right now, let's put it in drafT.